apps/vaporgui/MainForm.cpp

```diff
@@ -888,6 +892,11 @@ void MainForm::_createFileMenu()
     _dataImportMPAS_Action->setToolTip("Specify one or more MPAS output files to import into the "
                                        "current session");
 
+    _dataImportUGRID_Action = new QAction(this);
```

Comment:
There is a simplified method for adding data importer menu items. The DCP importer is an example.

Reply:
Done